src/jsc/jsc_value.hpp

```diff
@@ -85,6 +85,12 @@ inline bool jsc::Value::is_constructor(JSContextRef ctx, const JSValueRef &value
     return JSValueIsObject(ctx, value) && JSObjectIsConstructor(ctx, (JSObjectRef)value);
 }
 
+template<>
+inline bool jsc::Value::is_error(JSContextRef ctx, const JSValueRef &value) {
```

Comment:
whenever we change the abstraction layer we need to handle it both for Nodejs and RN

Reply:
Thanks for catching this - I thought about it and forgot about it :)